clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/requestqueue/QueuedRequestException.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.requestqueue;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.TOO_MANY_REQUESTS)
+public class QueuedRequestException extends RuntimeException {
```

Comment:
needed to move this to a RuntimeException instead of a subclass of TimeoutException which is checked in order to get the spring mvc exception handling stuff to work